azext_iot/central/providers/device_provider.py

```diff
@@ -246,3 +262,28 @@ def get_all_registration_info(
         ]
 
         return result
+
+    def get_registration_summary(self, central_dns_suffix="azureiotcentral.com"):
+        registration_summary = {}
+
+        devices = self.list_devices(central_dns_suffix=central_dns_suffix)
```

Comment:
FYI - this feature is currently going to be broken for apps with more than 25 devices since we don't yet support device paging. I think it might be a good exercise for you to try implement device paging as part of this PR.